src/components/form/Select/index.js

```diff
@@ -0,0 +1,20 @@
+import './styles.scss';
+
+const Select = ({ name, options, placeholder, register, handleFocus }) => {
+  return (
+    <div className="Select">
```

Comment:
maybe this has to be in lower case as the rest of the classnames of other components 🤔

Reply:
Agree